README.md

```diff
@@ -301,6 +302,40 @@ The model is evaluated automatically during [activities](#activity). You can che
 print(my_actor.temperature_in_K)
 ```
 
+#### Radiation Modelling
+PASEOS models three types of radiation effects.
+1. Data corruption due to single event upsets which a event rate $r_d$.
+2. Unexpected software faults leading to a random interruption of [activities](#activity) with a Poisson-distributed event rate $r_i$ per second
+3. Device failures with a Poisson-distributed event rate $r_f$ per second, which can be imputed mostly to single event latch-ups
+   
+You can add a radiation model with
+
+```py
+    from paseos import SpacecraftActor, ActorBuilder
+    my_actor = ActorBuilder.get_actor_scaffold("my_actor", SpacecraftActor, pk.epoch(0))
+    ActorBuilder.set_radiation_model(
+        actor=my_actor,
+        data_corruption_events_per_s=r_d,
+        restart_events_per_s=r_i,
+        failure_events_per_s=r_f,
+    )
+```
+
```

Comment:
Added a bit above that the user should specify the values for the device they are interested in. I don't want to give more details to avoid confusion. Anybody who has event numbers for this events should be aware of it anyways?